src/Hedgehog/Script.fsx

```diff
@@ -57,8 +58,8 @@ Gen.printSample genLeapYear
 
 // Fails due to integer overflow
 Property.print <| property {
-    let! x = Gen.int
-    let! y = Gen.int
+    let! x = Gen.int <| Range.constantBounded ()
+    let! y = Gen.int <| Range.constantBounded ()
```

Comment:
I want to hold off on this as long as possible until the path forward is clear.

I like the minimalism we have in the Haskell version (one combinator per type) and it's not clear to me what the default should necessarily be. In QuickCheck, the default for integers is equivalent to `Gen.int <| Range.linear 0 100`.

It's easy for people to experiment with their own default combinators by combining the primitives that we currently offer (i.e. `Gen.int` and the `Range` combinators)